packages/babel-plugin/src/utils/types.tsx

```diff
@@ -19,7 +26,7 @@ export interface SheetCssItem {
   css: string;
 }
 
-export type CssItem = UnconditionalCssItem | LogicalCssItem | SheetCssItem;
+export type CssItem = UnconditionalCssItem | ConditionalCssItem | LogicalCssItem | SheetCssItem;
```

Comment:
I think we will need to generate the flow types too.

Reply:
Looks like we only generate flowtypes for the `jest` and `react` packages, so these changes don't affect those types.